### PR TITLE
Main menu

### PR DESCRIPTION
**Pull request by albertJkt**
Add working connection string (win 10) and change DB a bit.

### CODE REVIEW

**Review by Moduxx — COMMENTED** [review]
Project ready 8)

File: CafeApp/CafeApp/AddRestaurant.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+namespace CafeApp
+{
+    public partial class AddRestaurant : Form
+    {
+        string name;
+        string address;
+        string phone;
+        int tables;
+        string workdays;
```
Comment (WarHatch) [inline CafeApp/CafeApp/AddRestaurant.cs:20]: Su tuo darbo laiku, man vis dar kliūna. Na bet čia jau dizaino reikalas

File: CafeApp/CafeApp/LoginForm.cs [hunk 2]
```diff
@@ -13,13 +13,12 @@ namespace CafeApp
 {
     public partial class LoginForm : Form
     {
-        SqlConnection connect = new SqlConnection(@"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=" + Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments) + @"\GitHub\cafe-demo\CafeApp\CafeApp\Database1.mdf;Integrated Security=True");
+        SqlConnection connect = new SqlConnection(@"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=C:\Users\Albert\Documents\GitHub\cafe-demo\CafeApp\CafeApp\Database1.mdf;Integrated Security=True");
```
Comment (WarHatch) [inline CafeApp/CafeApp/LoginForm.cs:16]: Why was this changed to a local DB?

File: CafeApp/CafeApp/RegisterWindow.cs [hunk 3]
```diff
@@ -15,8 +15,9 @@ public partial class RegisterWindow : Form
         public string password;
         public bool inserted=false;
 
-        SqlConnection connection = new SqlConnection(@"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename="+Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments)+@"\GitHub\cafe-demo\CafeApp\CafeApp\Database1.mdf;Integrated Security=True");
-       
+        SqlConnection connection = new SqlConnection(@"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=C:\Users\Albert\Documents\GitHub\cafe-demo\CafeApp\CafeApp\Database1.mdf;Integrated Security=True");
```
Comment (WarHatch) [inline CafeApp/CafeApp/RegisterWindow.cs:18]: Another change to local DB. Can we fix this?

**Review by WarHatch — COMMENTED** [review]
Local DB issue
